src/main/java/innohack/gem/entity/rule/rules/CsvHeaderColumnValue.java

```diff
@@ -5,13 +5,13 @@
 import innohack.gem.entity.rule.ParamType;
 import innohack.gem.entity.rule.Parameter;
 import innohack.gem.entity.rule.RuleType;
-import java.util.Collection;
+import java.util.List;
 
 public class CsvHeaderColumnValue extends Rule {
 
   static final String LABEL = "CSV Header Column Value";
   static final RuleType RULE_TYPE = RuleType.CSV;
-  static final Collection<Parameter> PARAMETERS =
```

Comment:
@yclim Noticed all the Collections are changed to List. We lose some flexibility in using either a List or Set... but which should be fine as mostly we are simply using List.

Reply:
true. I think if we need uniqueness in certain cases then we can later refactor those var to use Set ADT explicitly. Thought it might be more readable and less ambiguous this way.